aiomisc/pool.py

```diff
@@ -170,7 +170,7 @@ async def __release(self, instance: Any) -> None:
 
         self._instances.put_nowait(instance)
 
-    def acquire(self) -> ContextManager:
+    def acquire(self) -> AsyncContextManager[T]:
         return ContextManager(self.__acquire, self.__release)
```

Comment:
function has AsyncContextManager as expected output but there is just ContextManager. Some braincrush misnaming is here

Reply:
`ContextManager` it was not a generic `ContextManager`

Reply:
I was confused with such naming too.

Reply:
it's `aiomisc.pool.ContextManager` this should not be changed, since the module's namespace does not overlap with the standard library's namespace. Second, because of backwards compatibility.